well-architected/services/networking/azure-application-gateway.md

```diff
@@ -130,7 +130,7 @@ Azure Advisor helps you ensure and improve continuity of your business-critical
 
 ## Operational excellence
 
-Monitoring and diagnostics are crucial. Not only can you measure performance statistics but also use metrics troubleshoot and remediate issues quickly. We recommend you review the [Operational excellence design principles](../../devops/principles.md).
+Monitoring and diagnostics are crucial for ensuring operational excellence of your Application Gateway. Not only can you measure performance statistics but also use metrics to troubleshoot and remediate issues quickly. We recommend you review the [Operational excellence design principles](../../devops/principles.md).
```

Comment:
Not just for the health operation of the gateway, but also for the workload that it's fronting.  Can you expand this thought a bit more please?